tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "lib": ["es2017", "dom"],
-        "target": "es6",
+        "lib": ["es2020", "dom"],
+        "target": "es2019",
```

Comment:
Oskar will be so happy about this 🎉  No more transpiling promises to generator functions 🙏  

But do we not need to bump the node version in the Dockerfile for this to work in prod?

Reply:
The image we're using [node:12-alpine](https://hub.docker.com/layers/node/library/node/12-alpine/images/sha256-c29fba1f3ea0c23ca3ab59367dc644a594c8186fe28067ea4ac7e51b4a0956a1?context=explore) was upgraded to node version 12.19.0,[ it looks like >12.9.0 supports es2020.](https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12)